Feat: Allow users to exclude Replay package from Web (#4006)
getsentry/sentry-react-native: CHANGELOG.md

### Fixes

> - Exclude Sentry Web Replay by default, reducing the code in 130KB. ([#4006](https://github.com/getsentry/sentry-react-native/pull/4006))
> You can keep Sentry Web Replay by setting `includeWebReplay`as `true` on your metro config as shown on the snippet:
> ```js
>   // For Expo
>   const { getSentryExpoConfig } = require("@sentry/react-native/metro");
>   const config = getSentryExpoConfig(__dirname, { includeWebReplay: true });
> 
>   // For RN
>   const { getDefaultConfig } = require('@react-native/metro-config');
>   const { withSentryConfig } = require('@sentry/react-native/metro');
>   module.exports = withSentryConfig(getDefaultConfig(__dirname), { includeWebReplay: true });
> 
> ```
> 

Reviewer: Nit. Just moved some white spaces around.

````suggestion
- Exclude Sentry Web Replay by default, reducing the code in 130KB. ([#4006](https://github.com/getsentry/sentry-react-native/pull/4006))
  - You can keep Sentry Web Replay by setting `includeWebReplay` to `true` in your metro config as shown in the snippet:

  ```js
  // For Expo
  const { getSentryExpoConfig } = require("@sentry/react-native/metro");
  const config = getSentryExpoConfig(__dirname, { includeWebReplay: true });

  // For RN
  const { getDefaultConfig } = require('@react-native/metro-config');
  const { withSentryConfig } = require('@sentry/react-native/metro');
  module.exports = withSentryConfig(getDefaultConfig(__dirname), { includeWebReplay: true });
  ```

````
